KeeTrayTOTP.csproj

```diff
@@ -45,8 +45,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="KeePass">
-      <HintPath>C:\Program Files (x86)\KeePass Password Safe 2\KeePass.exe</HintPath>
```

Comment:
Hi Uwe, thanks for contributing! Removing the other hintpath's will break the build flow for other developers. Currently there is not a clean way of referencing `Keepass.exe` that would work for everyone. I am thinking of creating a nuget package for it, or ask Keepass's maintainer to do that in order to cleanly reference it.

I think it's fine if you leave yours be.

Reply:
I had commited that accidentally.
I have already undone the change in the .csproj see my second commit. 
We should leave the HintPath as it was.
I think every developer who has KeePass in another directory then the default directory should adapt it himself.